backend/src/test/java/gov/cdc/usds/simplereport/api/BaseApiTest.java

```diff
@@ -3,133 +3,107 @@
-import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.HttpStatus;
 
 @SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
 public abstract class BaseApiTest {
 
+  private static final Logger LOG = LoggerFactory.getLogger(BaseApiTest.class);
+
   protected static final String ACCESS_ERROR =
       "Current user does not have permission for this action";
 
   @Autowired private DbTruncator _truncator;
   @Autowired protected OrganizationInitializingService _initService;
+  @Autowired private TestDataFactory _dataFactory;
   @Autowired protected DemoOktaRepository _oktaRepo;
   @Autowired protected GraphQLTestTemplate _template; // screw delegation
-  @MockBean protected AuthorizationService _authService;
-  @MockBean protected IdentitySupplier _supplier;
-
-  private static final List<OrganizationRoleClaims> USER_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims("DIS_ORG", Set.of(OrganizationRole.USER)));
-  private static final List<OrganizationRoleClaims> OUTSIDE_USER_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims("DAT_ORG", Set.of(OrganizationRole.USER)));
-  private static final List<OrganizationRoleClaims> ADMIN_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DIS_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ADMIN)));
-  private static final List<OrganizationRoleClaims> OUTSIDE_ADMIN_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DAT_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ADMIN)));
-  private static final List<OrganizationRoleClaims> ENTRY_ONLY_ORG_ROLES =
-      Collections.singletonList(
-          new OrganizationRoleClaims(
-              "DIS_ORG", Set.of(OrganizationRole.USER, OrganizationRole.ENTRY_ONLY)));
+  @Autowired private DemoUserConfiguration _users;
+
+  private String _userName = null;
 
   protected void truncateDb() {
     _truncator.truncateAll();
   }
 
   protected void useOrgUser() {
-    LoggerFactory.getLogger(BaseApiTest.class).info("Configuring auth service mock for org user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(USER_ORG_ROLES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring auth service mock for org user");
+    _userName = TestUserIdentities.STANDARD_USER;
   }
 
   protected void useOutsideOrgUser() {
-    LoggerFactory.getLogger(BaseApiTest.class)
-        .info("Configuring auth service mock for outside org user");
-    when(_supplier.get()).thenReturn(TestUserIdentities.STANDARD_USER_ATTRIBUTES);
-    when(_authService.findAllOrganizationRoles()).thenReturn(OUTSIDE_USER_ORG_ROLES);
-    _initService.initCurrentUser();
+    LOG.info("Configuring auth service mock for outside org user");
+    _userName = "intruder@pirate.com";
```

Comment:
let's make all these other usernames constants in `TestUserIdentities` just like `STANDARD_USER` and `SITE_ADMIN_USER`